cassandane/Cassandane/Cyrus/JMAPCore.pm

```diff
@@ -870,4 +870,28 @@ sub test_blob_set_complex
     $self->assert_num_equals(length $target, $res->[2][1]{list}[0]{size});
 }
 
+sub test_blob_upload_repair_acl
+    :min_version_3_7 :needs_component_jmap :JMAPExtensions
+{
+    my $self = shift;
+    my $jmap = $self->{jmap};
+    my $admin = $self->{adminstore}->get_client();
+
+    $jmap->Upload("hello", "application/data");
+
+    my $file = abs_path('data/mime/testEmail.eml');
+    open(FH, "<$file");
```

Comment:
nit: 2 arg open and global file handles are generally discouraged these days. Preferably this would be:
```
open(my $fh, '<', $file);
local $/;
my $binary = <$fh>;
close($fh);
```

... though I realize we haven't standardized on this yet so there's a mix all over cassandane

Reply:
done